doc/release/0.4.rst

```diff
@@ -0,0 +1,104 @@
+#. Removed ``armi.utils.dochelpers``; not relevant to nuclear modeling. (`PR#1662 <https://github.com/terrapower/armi/pull/1662>`_)
+#. Removing old tools created to help people convert to the current database format: ``armi.bookkeeping.db.convertDatabase()`` and ``ConvertDB``. (`PR#1658 <https://github.com/terrapower/armi/pull/1658>`_)
+#. Removing the unused method ``Case.buildCommand()``. (`PR#1773 <https://github.com/terrapower/armi/pull/1773>`_)
+#. Removed the variable ``armi.physics.neutronics.isotopicDepletion.ORDER``. (`PR#1671 <https://github.com/terrapower/armi/pull/1671>`_)
+#. Removing extraneous ``ArmiOjbect`` methods. (`PR#1667 <https://github.com/terrapower/armi/pull/1667>`_)
+    * Moving ``ArmiObject.getBoronMassEnrich()`` to ``Block``.
+    * Moving ``ArmiObject.getPuMoles()`` to ``Block``.
+    * Moving ``ArmiObject.getUraniumMassEnrich()`` to ``Block``.
+    * Removing ``ArmiObject.getMaxUraniumMassEnrich.()``.
+    * Removing ``ArmiObject.getMaxVolume()`` & ``Block.getMaxVolume()``.
+    * Removing ``ArmiObject.getPuFrac()``.
+    * Removing ``ArmiObject.getPuMass()``.
+    * Removing ``ArmiObject.getPuN()``.
+    * Removing ``ArmiObject.getZrFrac()``.
+    * Removing ``ArmiObject.printDensities()``.
+    * Moving ``Composite.isOnWhichSymmetryLine()`` to ``Assembly``.
+    * Removing ``Block.isOnWhichSymmetryLine()``.
+#. Removing the ``Block.reactor`` property. (`PR#1425 <https://github.com/terrapower/armi/pull/1425>`_)
+#. Moving several ``ArmiObject`` methods. (`PR#1425 <https://github.com/terrapower/armi/pull/1425>`_)
+    * Moving ``ArmiObject.getNeutronEnergyDepositionConstants`` to ``Block``.
+    * Moving ``ArmiObject.getGammaEnergyDepositionConstants`` to ``Block``.
+    * Moving ``ArmiObject.getTotalEnergyGenerationConstants`` to ``Block``.
+    * Moving ``ArmiObject.getFissionEnergyGenerationConstants`` to ``Block``.
+    * Moving ``ArmiObject.getCaptureEnergyGenerationConstants`` to ``Block``.
+#. Removing the parameter ``rdIterNum``. (`PR#1704 <https://github.com/terrapower/armi/pull/1704>`_)
+#. Removing the parameters ``outsideFuelRing`` and ``outsideFuelRingFluxFr``. (`PR#1700 <https://github.com/terrapower/armi/pull/1700>`_)
+#. Removing the setting ``doOrificedTH``. (`PR#1706 <https://github.com/terrapower/armi/pull/1706>`_)
+#. Changing the Doppler constant params to ``VOLUME_INTEGRATED``. (`PR#1659 <https://github.com/terrapower/armi/pull/1659>`_)
+#. Change ``Operator._expandCycleAndTimeNodeArgs`` to be a non-static method so it no longer needs to take arbitrary args. (`PR#1766 <https://github.com/terrapower/armi/pull/1766>`_)
+#. Database now writes state at the last time node of a cycle rather than during the `DatabaseInterface.interactEOC` interaction, making this node accessible since its the more useful node, and at a unique time. (`PR#1090 <https://github.com/terrapower/armi/pull/1090>`_)
```

Comment:
```suggestion
#. Database now writes state at the last time node of a cycle rather than during the ``DatabaseInterface.interactEOC`` interaction, making this node accessible since its the more useful node, and at a unique time. (`PR#1090 <https://github.com/terrapower/armi/pull/1090>`_)
```